models/meshery_remote_provider.go

```diff
@@ -198,21 +198,21 @@ func (l *MesheryRemoteProvider) GetUserDetails(req *http.Request) (*User, error)
 func (l *MesheryRemoteProvider) GetSession(req *http.Request) error {
 	ts, err := l.GetToken(req)
 	if err != nil {
-		err = errors.Wrap(err, "Error: unable to get session")
-		logrus.Error(err)
+		err = fmt.Errorf("Session not found")
```

Comment:
Same over here, `logrus.Error` can be used directly.

Reply:
`fmt.Error` does not exist, only logrus has `Errorf`

Reply:
Logurs.fatal as used to maintain consistency.